modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -214,6 +216,55 @@ The `IF NOT EXISTS` and `OR REPLACE` parts of these commands cannot be used toge
 ====
 
 
+==== Create databases using seeding options
+
+Neo4j has built-in support for a seed from a mounted file system (file), FTP server, HTTP/HTTPS server, Amazon S3, Google Cloud Storage, and Azure Cloud Storage.
+To select a seed provider, you need to configure xref:configuration/configuration-settings.adoc#config_dbms.databases.seed_from_uri_providers[dbms.databases.seed_from_uri_providers].
+
+For more information on seeding from URI, refer to xref::clustering/databases.adoc#cluster-seed-uri[Manage databases in a cluster -> Seed from URI].
```

Comment:
I would rather put this sentence at the end of the section after the example and the last sentence below and say something more like,
```suggestion
For more details and examples of seeding from URI, refer to xref::clustering/databases.adoc#cluster-seed-uri[Manage databases in a cluster -> Seed from URI].
```